### DISCUSSION TITLE
Welcome to hass-amber-electric Discussions!

### DISCUSSION THREAD

**Discussion by davewatson91** [davewatson91/hass-amber-electric#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by bengAThome** [comment]
Thanks for starting this fork. I'll help any way I can :)
